empirical-org/Empirical-Core
# proofreader main interface accessibility (#6255)

* update package version for marking logic

* quill-marking-logic security upgrades (#6194)

* pushing webpack-dev-server and kind-of changes

* update kind-of package and take out webpack errors

* testing changes

* fix broken tests

* merged w qcl fixes

* take out force-resolution

* emilia's suggestions

* changed tests to pull from the right file

* wip

* wip

* wip

* get computed width stuff working more consistently

* finish everything but the review parts

* differentiate types of unnecessary edits

* more specificity in methods

* wip

* make sure tooltip is properly aligned on first load

* scrollable review modals

* hopefully final updates

* more fixes

* making sure save and exit works as expected and adjusting scroll height

* update package

* fix tests and get rid of screenreader extension conflict

* update package.lock

* revisions

* switch to using contenteditable for proofreader

* make sure review works on modal and remove fake explanation stuff

* try making middle section shorter still

* try it like this

* change whole edit tooltip height instead

* fix linter errors

* one more linter fix

* more revisions

* adjust mobile height again

* fixed it

* one last change

Co-authored-by: Eric Adams <erictayloradams@gmail.com>
Co-authored-by: cissyxyu <57366100+cissyxyu@users.noreply.github.com>
Co-authored-by: dandrabik <dandrabik@users.noreply.github.com>
Co-authored-by: Cissy Yu <cissy@quill.org>
Co-authored-by: Anathomical <thomas.robertson@quill.org>

## WHAT

## WHY

## HOW

## Screenshots
(If applicable. Also, please censor any sensitive data)

## Have you added and/or updated tests?
(The answer should mostly be 'YES'. If you answer 'NO', please justify.)

## Have you deployed to Staging?
(Possible answers: YES, Not yet - deploying now!, NO - non-app change, NO - tiny change)
